composer.json

```diff
@@ -8,7 +8,7 @@
     "sort-packages": true,
     "autoloader-suffix": "csm",
     "platform": {
-      "php": "7.3"
+      "php": "5.6"
```

Comment:
Let's make these composer changes as a separate PR in order to keep the history more clean.

Reply:
See https://github.com/GoogleChromeLabs/wp-sitemaps/pull/30